tests/containers/docker_runc.pm

```diff
@@ -24,43 +25,82 @@ sub run {
     select_serial_terminal;
 
     my ($running_version, $sp, $host_distri) = get_os_release;
-    my $docker = $self->containers_factory('docker');
-    my @runtimes = ();
-    push @runtimes, "runc" if (is_leap(">15.1") or !is_sle('=15'));
+    my $runc = "runc";    # runc executable
 
-    record_info 'Setup', 'Setup the environment';
-    # runC cannot create or extract the root filesystem on its own. Use Docker to create it.
-
-    # create the rootfs directory
+    # Runtime setup and installation
+    record_info 'Test #1', 'Installation and test preparation';
+    if (script_run("which $runc") != 0) {
+        if (is_transactional) {
+            select_console 'root-console';
+            trup_call("pkg install $runc");
+            check_reboot_changes;
+        } else {
+            zypper_call("in $runc");
+        }
+    }
```

Comment:
You can now `use containers::common "install_packages"` like this:
```suggestion
    install_packages($runc);
```

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/containers/common.pm#L353

Reply:
oh nice, that simplifies this whole block!